examples/pom.xml

```diff
@@ -98,143 +98,145 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-server</artifactId>
```

Comment:
This diff is a bit hard to track - is the main thing that you removed jetty server? Any idea why we had this before?